### DISCUSSION TITLE
What are GB-Hrs?

### DISCUSSION THREAD

**Discussion by cobrexus** [vercel/vercel#5627] [Help] [answered]
The [typical monthly usage guidelines in the Vercel fair use policy](https://vercel.com/docs/platform/fair-use-policy#typical-monthly-usage-guidelines) have this table:

![image](https://user-images.githubusercontent.com/34424160/103482037-589daa80-4dd6-11eb-9603-6b15a65a0afa.png)

---

I have two questions:

1. What are **GB-Hrs** for serverless function execution? How are they measured?
2. The callout says *Hobby accounts are restricted to non-commercial personal use only. All commercial usage of the platform requires either a Pro or Enterprise plan.* Does this mean that a monetised blog needs to be on a Vercel Pro plan?

Thanks in advance

**Comment by cobrexus** [comment]
*bump*

I'd be grateful if somebody could shed some light on this.

**Comment by meotimdihia** [comment, marked as answer]
I asked this question and got the answer: 
https://github.com/vercel/vercel/discussions/5384

**Reply by cobrexus** [reply]
Thanks for answering!

You mention on the linked post that you changed the amount of memory allocated, I'd be grateful if you share some advice on https://github.com/vercel/vercel/discussions/5682, thanks in advance :)

**Reply by meotimdihia** [reply]
If you use next.js, It required 1024MB for the build based on my experience - even though, the serverless function was required lower memory
![chrome_2021-01-12_08-33-33](https://user-images.githubusercontent.com/300961/104257898-05e88080-54b1-11eb-87ae-7e3521750353.png)

Note: I removed config memory for functions. If you set a wrong memory value, Vercel can't build the serverless function and still mark it as successful - You can try and check it.
